src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -565,6 +565,11 @@
           "type": "string"
         },
 
+        "id": {
+          "description": "A unique identifer for the result.",
+          "type": "string"
+        },
+
```

Comment:
I tend to think 'id' could be purposed in several ways, so it's fine to remain general. We also have a log 'id' member and I tend to think result.id name should match that.